### ISSUE TITLE
Implement: Multimaterial prints - Using AMS together with External spools.

### ISSUE THREAD

**Issue by NVNDO** [bambulab/BambuStudio#3902]
To this date it's "impossible" to print multimaterial using the AMS together with external spools.

Here are the following "easy" ways (one would imagine it to work - but none of them do)

Combining AMS with external Spools (Split body): 
![Filament change 3](https://github.com/bambulab/BambuStudio/assets/73851760/0d96c96b-dd70-4762-b124-f1a9eb641d6f)
Problem: Does straight up not work. FAIL!

Using Filament-change (Solid body): (with external spools and deactivated AMS)
![Filament change](https://github.com/bambulab/BambuStudio/assets/73851760/bd253cd0-2540-44bd-90d6-c5779a6b8816)
Problem: The filament change command is always "ignored" - FAIL!

Using Pause (Solid body) to then do a manual Unload/ Load of the desired filament: 
(The only way to actually combine external filaments / or combine if not AMS-compatible filaments are used)
![Filament change](https://github.com/bambulab/BambuStudio/assets/73851760/e3699cf8-5f31-4038-abca-0e143d796df8)
Problem: The same printing-profile will be used throughout the entire print - FAIL!

Using only External spools (Spilt body):
![Filament change 4](https://github.com/bambulab/BambuStudio/assets/73851760/68038b5e-3134-4947-8e51-322c1b2d6d15)
Problem: No Pause will be triggered to change the filament! - FAIL!

**Describe the solution you'd like**

1. Being able to combine AMS with external spools. 
Approach: Trigger a message for manual filament-change (Unload/Load) instead of activating the AMS motors.
(Don't forget to implement the change of filament-profile)

2. Being able to combine External spools.
![Filament change 4](https://github.com/bambulab/BambuStudio/assets/73851760/68038b5e-3134-4947-8e51-322c1b2d6d15)
(same screenshot as above) 
Approach: Trigger a manual filament change (Unload/Load) and use each color with its given preset (filament-profile)

**Describe alternatives you've considered**
- Creating an in-between profile to fit somehow both filaments-types (just not possible)
- Using alternative G-code to trigger a temperature change together with pause (too complicated - ruins the entire experience of an otherwise simple workflow)

**Additional context**
Implement: Multimaterial prints - Using AMS together with External spools. AND combining external filaments.

**Comment by OzDruiD** [comment]
100% - we need this, +++  upvote!

**Comment by momentmal** [comment]
+1

**Comment by Bokbacken** [comment]
Would love to see this!

**Comment by brentbrooks70** [comment]
Trying to do this just now. wish it was already a thing. 
Absolutely a needed feature!!
+1

**Comment by DIYtechie** [comment]
Come on. This should be possible!

**Comment by neward99** [comment]
I am really disappointed that this was not implemented

**Comment by bchevreau** [comment]
How are we 1 year later and this is not in Bamby studio? What is the point of having an external spool, where it has to be manually loaded anyways, if we can't complement our AMS with... the external spool available in Bambu studio. We need to be able to just have the slicer add g-code to trigger a manual swap. This is already the case for things like empty filament and such, there is nothing to implement, just make it work please... If I have to unload my AMS (beyond the obvious multi-material limitations of the print profile itself) mid-print to change a color, or unload beforehand to add the spool that's outside, what's the point of providing that spool in the slicer in the first place. Please fix this !

**Comment by NVNDO** [comment]
Yep! No updates so far.. and I doubt that Bambu has even looked at this feature-request.
I basically threw all my projects that required this over board and called it a day. 

Though I hope that one day - probably when I don't need it anymore -´they'll implement this.

**Comment by omitsch** [comment]
It's crazy, that this is not a feature yet

**Comment by IMBACalimba** [comment]
And people still waiting for this basic "feature" to be implemented :-/

**Comment by HarmlessFun69** [comment]
YES PLEAAASEEE

**Comment by NuiQuant** [comment]
Still need this

**Comment by young-hnau** [comment]
Still need this!!!

**Comment by HarmlessFun69** [comment]
Bump !

**Comment by Fangio92** [comment]
Bump!

**Comment by hungnguyen7** [comment]
Bump!!!!
